Gemfile

```diff
@@ -11,3 +11,6 @@ gem 'minitest', '5.10.1'
 group :tools do
   gem 'github_changelog_generator', '~> 1.12.0'
 end
+
+# Added at 2017-12-06 10:20:41 -0500 by cwolfe:
+gem "byebug", "~> 9.1"
```

Comment:
I love this!

May need to remove it when we merge to core InSpec, but in the mean time :+1:

Reply:
Ahh, that was actually mistakenly committed.  If we add byebug, it should be in a `development` group or something.